torch_tvm/operators.cpp

```diff
@@ -444,6 +444,16 @@ RegisterTVMOperator reg({
        auto out = tvm::relay::CallNode::make(op, inputs, tvm::Attrs(), {});
        return out;
      }},
+    {Symbol::fromQualString("aten::clamp"),
+     [](Node* node, tvm::Array<tvm::relay::Expr> inputs) {
+       auto op = tvm::relay::Op::Get("clip");
+       auto clip_attrs = tvm::make_node<tvm::relay::ClipAttrs>();
+       clip_attrs->a_min = relayToConstant<float>(inputs[1]);
```

Comment:
How do we know the second and third input is constant?

Reply:
At this point all scalars are make into `tvm::relay::ConstantNode` and I dont think they are part of `ArgumentSpec`, although I am not sure. So you can argue that two calls of `aten::clamp` can come with two different values of clamp and we certainly won't catch that.
But in the JIT graph you can check if those inputs come from `prim::Constant` nodes or not. That would be a good thing to add.

Reply:
@kimishpatel It makes more sense to me. Do we actually do this for other ops?

Reply:
we do `relayToConstant` everywhere, so it would be good to add this test. I will put it up for TODO.

Reply:
Should this diff depend on adding those `prim::Constant` checks, then?

Reply:
I dont think so.